wdl/ExternalWholeGenomeReprocessing.wdl

```diff
@@ -5,8 +5,6 @@ import "CopyFilesFromCloudToCloud.wdl" as Copy
 
 workflow ExternalWholeGenomeReprocessing {
 
-  String pipeline_version = "1.2"
```

Comment:
This was here since we thought we should give the adapter wdls same version as their scientific counterparts.

Reply:
Why?

Reply:
If you submit this external workflow without having workflow versions, won't people ask which semantic version is this? I also agree having the hash is enough info this is more like redundant version that we have to maintain overtime.

Reply:
The subworkflow WDL is part of the Cromwell input.
The release version is there if people want it.
I don't think a semantic version adds anything here,
and we would have to maintain it somehow.